### PR TITLE
aoscc/2024-guide: correct airport bus info, expand bus section, wording

### PR DESCRIPTION
**Pull request by TechCiel**

### CODE REVIEW

File: content/community/aoscc/2024-guide.zh.md [hunk 1]
```diff
@@ -82,33 +82,41 @@ tags = ["events"]
...
+吉林大学由于校园庞大，且相对偏远，公交线路并不是太好用，前往红旗街等繁华地段可以考虑网约车，约 15-20 元，20-30 分钟。
 
-北门外吉大南校站：可乘坐 G195 路、Z315 路、高新 1 号线、2 号线。
+附近的主要公交线路有：
 
-其中 G195 路可前往长春西站（站前街），或两站后换乘 G239 前往红旗街、长影博物馆、南湖公园西侧，Z315 路可前往桂林路/同志街、南湖公园东侧。
+- 高新开发区站（酒店 700m/南门 850m）：可乘坐 **G13** 路、**G222** 路，均为始发站
+- 前进大街站/星河街站（分别距酒店/南门 300m）：可乘坐 **G193** 路
+- 吉大南校站（距**北门** 100m）：可乘坐 **G195** 路、**Z315** 路，均为始发站
+- 吉大南校（东门）站（距**东门** 100m）：可乘坐 **G13** 路、**G222** 路、**Z315** 路，均距始发站两站
```
Comment (billchenchina) [inline content/community/aoscc/2024-guide.zh.md:94]: 站名不存在？但是好像和上面的重名了？

File: content/community/aoscc/2024-guide.zh.md [hunk 2]
```diff
@@ -82,33 +82,41 @@ tags = ["events"]
...
+        - **G239** 路（电台街站，朝阳方向）可前往**红旗街商圈**、**长影**旧址博物馆、**南湖公园**西侧
+- **Z315** 路：北门/东门出发，可前往**桂林路**/同志街商圈、**南湖公园**东侧
 
 ## 轨道交通
 
-长春市在运轨道交通线路 6 条，其中 1/2/6 号线为地铁，3/4/8 号线为轻轨，大部分站内换乘，阶梯票价一般 2-5 元，均支持交通联合。
+长春市在运轨道交通线路 6 条，其中 1/2/6 号线为**地铁**，3/4/8 号线为**轻轨**，大部分站内换乘，阶梯票价一般 **2-5 元**，均支持**交通联合**。
 
-乘坐 1/3/4 号线可到达长春站，2/6 号线可到达长春西站。其中 3 号线两侧分别可到达伪满皇宫、净月潭国家森林公园。
+乘坐 1/3/4 号线可到达**长春站**，2/6 号线可到达**长春西站**。其中 3 号线两侧分别可到达**伪满皇宫**、**净月潭**国家森林公园。
```
Comment (billchenchina) [inline content/community/aoscc/2024-guide.zh.md:113]:
```suggestion
乘坐 1/3/4 号线可到达**长春站**，2/6 号线可到达**长春西站**。其中 3 号线起终点分别可到达**伪满皇宫**、**净月潭**国家森林公园。
```

Comment (billchenchina) [reply content/community/aoscc/2024-guide.zh.md:113]: 3 号线终点是长影世纪城吧

**Review by billchenchina — CHANGES_REQUESTED** [review]

File: content/community/aoscc/2024-guide.zh.md (hunk 1 above)
Comment (TechCiel) [reply content/community/aoscc/2024-guide.zh.md:94]: 我明天去现场看看到底叫什么站。

Comment (TechCiel) [reply content/community/aoscc/2024-guide.zh.md]: fixed

**Review by billchenchina — APPROVED** [review]
lgtm, thank you!

**Review by MingcongBai — APPROVED** [review]
Incorporated review from @KexyBiscuit to clarify that attendees may also grab their swags at the venue.

**Comment by MingcongBai** [comment]
Very nice guide!

**Pull request merged**